run-tests.sh

```diff
@@ -1,16 +1,22 @@
-#!/bin/sh
+#!/bin/bash
 #
 # This file is part of REANA.
 # Copyright (C) 2017, 2018 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+
+cli_docs_url=https://raw.githubusercontent.com/reanahub/docs.reana.io/master/docs/reference/reana-client-cli-api/index.md
+
 pydocstyle reana_client && \
 isort -rc -c -df **/*.py && \
 reana-client --help > cmd_list.txt && \
 diff -q -w docs/cmd_list.txt cmd_list.txt  && \
 rm cmd_list.txt && \
+python scripts/generate_cli_api.py > cli_api.md && \
+diff -q -w  cli_api.md <(curl -s $cli_docs_url) && \
```

Comment:
What do you think about improving the message so when we read the Travis failure (for example [this one](https://travis-ci.org/github/reanahub/reana-client/jobs/675244135#L626)) we don't get something like `Files cli_api.md and /dev/fd/63 differ` but something more descriptive?
```bash
... && \
$docs_differ_error_msg='Current reana-client differs with the documentation. Please update http://docs.reana.io/reference/reana-client-cli-api/.'
(diff -q -w  cli_api.md <(curl -s $cli_docs_url) || (echo $docs_differ_error_msg && exit 1)) && \
...
```